### DISCUSSION TITLE
如何配置config文件连接 coze api

### DISCUSSION THREAD

**Discussion by Alex-ZhaoR** [zhayujie/chatgpt-on-wechat#2100] [Q&A]
目前通过调整config文件能够支持 接入openapi 或者 linkai等，如何能够支持使用 coze（扣子）平台的api呀，这样就可以把coze的智能体和微信连接起来了

**Comment by space624** [comment]
使用coze2openai做一个中转站就好了,然后chatgpt-on-wechat监听这个中转站
